build/__javaVersions.sh

```diff
@@ -1,4 +1,4 @@
 # This file was auto-generated from 'constants.yaml'. Changes may be overridden.
```

Comment:
For my own personal inventory, this PR is adding the following platform versions:

- Java 17.0.2
- Node 16.14.2
- PHP 7.4.28 and 8.1.4
- Ruby 3.0.0 and 3.1.1

I see some versions are added to buster and focal as well (such as Node 14.19.1) -- just curious if these versions were previously meant to be added to buster and focal and weren't, or if there's a separate reason we're adding them now.

Reply:
These versions are auto added to stretch, buster and focal when we triggered the platform binary pipelines. For each SDK we have these three kinds so that users can use it from different types of images.